src/main/java/Interfaces/IAction.java

```diff
@@ -0,0 +1,5 @@
+package Interfaces;
+
+public interface IAction {
+    String action();
```

Comment:
Thats not clear. You should document it or use a better naming. 
Can you tell me why Action should lead to some String? Thats smth i can't really understand looking at this exact line, and making me looking for other files to understand this exact line is not correct from your side.